client/src/api/schema/schema.ts

```diff
@@ -5997,6 +5997,87 @@ export interface components {
              */
             visible: boolean;
         };
+        /**
+         * HDAInaccessible
+         * @description History Dataset Association information when the user can not access it.
+         */
+        HDAInaccessible: {
+            /**
+             * Accessible
+             * @description Whether this item is accessible to the current user due to permissions.
+             */
+            accessible: boolean;
```

Comment:
This should be a const of `false` (set it to `Literal[False]` in the schema), or, if it is part of all models, should move into one of the base classes.

Reply:
that makes sense to me but maybe more needs to be done since this change types the client schema as `accessible: Record<string, never>;`

Reply:
I checked this out and the relevant generated OpenAPI schema seems fine:

```yaml
HDAInaccessible:
      description: History Dataset Association information when the user can not access it.
      properties:
        accessible:
          const: false
          title: Accessible
```

This is something we probably can fix since we are handling `const` cases in https://github.com/galaxyproject/galaxy/blob/dev/client/openapi_to_schema.mjs I can give it a try.

Interestingly enough, it does work when the const value is `true` but not with `false` :laughing: 

https://github.com/galaxyproject/galaxy/blob/cd779a47ad895a376314888b621a88f5abe1f6c2/client/src/api/schema/schema.ts#L2678

Reply:
Yeah, I think this might get fixed with @dannon's work on updating to openapi-fetch

Reply:
I fixed the boolean constant conversion for now in https://github.com/galaxyproject/galaxy/pull/17818/commits/7961b475bbd80ceb8bdd615b1e6eb0c67a9ff2a1